data/naltok ships.txt

```diff
@@ -369,7 +369,7 @@ ship "Grouper"
 	attributes
 		"capture class" "xochira novelty"
 		category "Utility"
-		"cost"
+		"cost" 23100000
```

Comment:
```suggestion
		"cost" 8400000
```
You're right that this needs a cost, but I think 23 million is a lot higher than it should be.

Reply:
Yeah, I just glanced over the stats. 8m looks way better